spec/requests/api/motorcycles_spec.rb

```diff
@@ -0,0 +1,95 @@
+require 'swagger_helper'
+
+RSpec.describe 'Motorcycles', type: :request do
+  path '/api/mototorcycles' do
+    get('list motorcycles') do
+      response(200, 'successful') do
+        after do |example|
+          example.metadata[:response][:content] = {
+            'application/json' => {
+              example: JSON.parse(response.body, symbolize_names: true)
+            }
+          }
+        end
+        run_test!
+      end
+    end
+
+    post('create motorcycle') do
+      response(200, 'successful') do
+        after do |example|
+          example.metadata[:response][:content] = {
+            'application/json' => {
+              example: JSON.parse(response.body, symbolize_names: true)
```

Comment:
Please add the user_id parameter for the Motorcycles post method. Also, add the schema properties for the post method.

example:
```
parameter name: 'user_id', in: :path, type: :number, description: 'user_id', schema: {
        type: :object,
        properties: {
          motor: {
            model: { type: :string },
            image: { type: :string },
            price: { type: :integer },
            description: { type: :text },
            duration_months: { type: :integer }
          }
        }
      }
```